src/python/WMCore/MicroService/MSTransferor/DataStructs/Workflow.py

```diff
@@ -484,3 +484,25 @@ def getOpenRunningTimeout(self):
         :return: an integer with the amount of secs
         """
         return self.data.get("OpenRunningTimeout", 0)
+
+    def getInputData(self):
+        """
+        Returns all the primary and parent data that has to be locked
+        and transferred with Rucio
+        :return: a list of unique blocks
+        """
+        return self.getChunkBlocks()
+
+    def getRucioGrouping(self):
+        """
+        Returns the rucio rule grouping to be defined for in primary
+        and/or parent input data placement, where:
+            * ALL: all blocks are placed under the same RSE
+            * DATASET: blocks can be scattered in multiple RSEs
```

Comment:
Please make a note here stating that we are talking about Rucio `DATASET` rather than a CMS `DATASET`